packages/styles/scss/patterns/cards-without-images/index.scss

```diff
@@ -0,0 +1,8 @@
+@import '../../globals/imports';
+@import './cards-without-images';
+@import './cards-without-images-group';
+
+.#{$prefix}--cards-without-images--g10 {
+  @include cards-without-images;
+  @include cards-without-images-group;
+}
```

Comment:
@sannminwin not sure if this captures the theming modifier. This is still going to render as a parent class and not a modifier class. A theme mixin should also be used to target only the specific theming tokens so that you aren't replicating all of the CSS code that doesn't need to be replicated.

cc: @annawen1

Reply:
fixed

Reply:
looks like you have to create mixins for both the `cards-without-images` and `cards-without-images-group` since they both use color tokens. 

@jeffchew if anything, maybe we revert back to what sann had before with the theme wrapper and merge. Then create a ticket to handle the theme modifier class to do later?